src/tests/components/Auth/ButtonSubmit/index.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import FormButton from 'components/Auth/ButtonSubmit';
+
+describe('When FormButton component is mounted', () => {
+  const label = 'Log in';
+
+  it('renders button with submit type', () => {
+    const { getByTestId } = render(<FormButton label={ label } />);
+    const formButton = expect(getByTestId('form-button-test'));
```

Comment:
How about `getByRole('button', )` instead of `getByTestId` ? IMO it's easier to understand and make the element query as close as how the user see it 💭

Reply:
Changed!